TileStache/Goodies/VecTiles/sort.py

```diff
@@ -32,7 +32,7 @@ def _by_scalerank(feature):
 def _by_population(feature):
     wkb, properties, fid = feature
     value_for_none = -1000
-    population = properties.get('population', value_for_none)
+    population = int(properties.get('population', value_for_none))
```

Comment:
We have a function in [transforms](https://github.com/mapzen/TileStache/blob/5e48fb9e6e8626bbbed081d1ccc16f1fd46a9c31/TileStache/Goodies/VecTiles/transform.py#L11-L19) that's geared towards parsing floats. I was thinking that it doesn't make sense here because population should be an integer, but on second thought we might want to re-use that and if we got a float back (and not `None`), coerce to an integer.